src/apps/yearn/ethereum/yearn.governance.contract-position-fetcher.ts

```diff
@@ -19,11 +18,11 @@ const appId = YEARN_DEFINITION.id;
 const groupId = YEARN_DEFINITION.groups.governance.id;
 const network = Network.ETHEREUM_MAINNET;
 
-@Register.ContractPositionFetcher({ appId, groupId, network })
 export class EthereumYearnGovernanceContractPositionFetcher extends SingleStakingFarmTemplateContractPositionFetcher<SynthetixRewards> {
   appId = appId;
   groupId = groupId;
   network = network;
+  groupLabel = 'Governance Staking';
```

Comment:
Spoke about this offline; we'll remove this on this PR, make it a required prop on the next PR, and then put it EVERYWHERE.